1-table.sql

```diff
@@ -0,0 +1,26 @@
+use IdfDB
+go
+drop table if exists Soldier
+go
+create table dbo.Soldier(
+	SoldierID int not null identity primary key,
+	FirstName varchar(25) not null
+		constraint c_Soldier_first_name_cannot_be_blank check(FirstName <> ''),
+	LastName varchar(25) not null
+		constraint c_Soldier_last_name_cannot_be_blank check(LastName <> ''),
+	SSN varchar(10) not null
+		constraint c_Soldier_SSN_cannot_be_blank check(SSN <> '')
```

Comment:
SSN must be in specific format. 8 numbers - 1 number. Example: 12345678-9 as shown in data.

Reply:
I added a constraint